### ISSUE TITLE
I can't check my Play integrity

### ISSUE THREAD

**Issue by hiru12** [1nikolas/play-integrity-checker-app#20]
![Screenshot_20231115_221343_Play Integrity API Checker](https://github.com/1nikolas/play-integrity-checker-app/assets/74873448/d2ed581e-83ad-4e46-ba39-fd32affa9337)
EEverytime I try to check this error pop ups.

**Comment by 1nikolas** [comment]
No idea why this happens. Does it happen every time?

**Comment by MeguminSama** [comment]
![Screenshot_20231117-005918.png](https://github.com/1nikolas/play-integrity-checker-app/assets/27697325/47ce3083-41de-487d-ad72-5c86bcef990a)

+1

**Comment by hiru12** [comment]
> No idea why this happens. Does it happen every time?

Yes. I've never been able to check my safety net/play integrity. I actually don't know if I passed the test or not.

**Comment by h1dd3n3y3** [comment]
> ![Screenshot_20231117-005918.png](https://user-images.githubusercontent.com/27697325/283636678-47ce3083-41de-487d-ad72-5c86bcef990a.png)
> 
> +1 happens every time

Same here

**Comment by Andreychik32** [comment]
Same issue for me.

**Comment by whyakari** [comment]
same here.

**Comment by MaxMacNeill-UNB** [comment]
+1

**Comment by BadBimr** [comment]
Same here.

**Comment by clayissimo** [comment]
Same here.

**Comment by MeguminSama** [comment]
It's working again now.

**Comment by RafaeloxMC** [comment]
"The calling app..." just means that the daily request limit for the API key exceeded. That's basically everything. Wait a day and it'll be fine.

**Comment by hiru12** [comment]
> "The calling app..." just means that the daily request limit for the API key exceeded. That's basically everything. Wait a day and it'll be fine.

My error is different from the other here.

It says "There was a transient error in the clients service."

**Comment by RafaeloxMC** [comment]
> > "The calling app..." just means that the daily request limit for the API key exceeded. That's basically everything. Wait a day and it'll be fine.
> 
> My error is different from the other here.
> 
> It says "There was a transient error in the clients service."

Im sorry, my reply was directed to @h1dd3n3y3 ([See this answer](https://github.com/1nikolas/play-integrity-checker-app/issues/20#issuecomment-1815569132))
